tests/core/process.c

```diff
@@ -713,11 +741,14 @@ store_export_address_if_mach_msg (const GumExportDetails * details,
 static gpointer
 sleeping_dummy (gpointer data)
 {
-  volatile gboolean * done = (gboolean *) data;
+  struct _TestThreadSyncData * sync_data = (struct _TestThreadSyncData *)data;
```

Comment:
- Always use typedefs.
- Missing space after cast. Cast should be dropped, though, as this is C (C++ would be a different story).

Reply:
I left the cast as you use the exact same construction in this file (and I prefer to use casts :D)